tests/Unit/Helpers/Domain/Structure/NeighborHelpers.cpp

```diff
@@ -464,7 +464,9 @@ std::vector<Neighbors<Dim>> valid_neighbors(
   const OrientationMap<Dim> aligned{};
   const size_t block_id = element_id.block_id();
   const size_t neighbor_block_id =
-      (face_type == FaceType::Block ? block_id + 1 : block_id);
+      (face_type == FaceType::Block
+           ? block_id + 2 * normal_dim + (side == Side::Lower ? 1 : 2)
+           : block_id);
```

Comment:
There are valid domains where the neighbor blocks in two directions are the same.  Can the AMR code handle that?

Reply:
Yes.  The reason I don't allow it in the test is that I would have to add an extra step when creating the valid_neighbor_flags that ensure that a given element only occurs once and be a valid choice in all directions it is a neighbor.  In the AMR code, these are conditions are guaranteed to be satisfied by the passed in `neighbor_flags`.  So it is a choice to not make an already complicated test more complicated.   I can test an explicit example to the test if you wish.

Reply:
OK, thanks for explaining.  I think it's fine as-is, then.